packages/kbn-expandable-flyout/src/components/index.test.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Panel } from '../models/panel';
+import { ExpandableFlyout } from '../..';
+import { LEFT_SECTION, PREVIEW_SECTION, RIGHT_SECTION } from './test_ids';
+import { ExpandableFlyoutContext } from './context';
+
+describe('ExpandableFlyout', () => {
+  it('should render right section', () => {
+    const context: ExpandableFlyoutContext = {
+      panels: {
+        right: {
+          id: 'key',
+        },
+        left: {},
+        preview: [],
+      },
+    } as unknown as ExpandableFlyoutContext;
```

Comment:
nit: could move this to some kind of `mocks` file here :)

Reply:
the 3 tests have slightly different values, and I kinda like having everything in the test, makes it easier to read (to me)